### PR TITLE
Ma jovanovic

### PR DESCRIPTION
**Pull request by BorisJov**
Integrating the Privacy Control Service

### CODE REVIEW

File: .github/workflows/docker-CI.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Docker Build and Push
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [ master, develop, ma_jovanovic ]
```
Comment (lakhoune) [inline .github/workflows/docker-CI.yml]: Can be removed right?

**Review by lakhoune — COMMENTED** [review]
It seems to me like there were some checks removed in whether we are using blockchain verification. Don't we need those anymore? 
What about the check on whether to use the privacy control service or not that we discussed last time?

File: .github/workflows/docker-CI.yml (hunk 1 above)
Comment (BorisJov) [reply .github/workflows/docker-CI.yml]: True

Comment (BorisJov) [reply .github/workflows/docker-CI.yml]: About the blockchain verification: it is now being done by the PCS, so no need for the old system.

Comment (BorisJov) [reply .github/workflows/docker-CI.yml]: There is a new check now: usesPrivacyControl

**Review by lakhoune — APPROVED** [review]

**Pull request merged**